steps/fetch-infisical-secrets/1.0.0/step.yml

```diff
@@ -0,0 +1,65 @@
+title: Fetch Infisical Secrets
+summary: |
+  Allows the collection of secrets from your Infisical account.
+description: |
+  Allow the collection of secrets from your Infisical account and injects them into your workflow in a secure manner.
+website: https://github.com/Willt123/bitrise-step-fetch-infisical-secrets
+source_code_url: https://github.com/Willt123/bitrise-step-fetch-infisical-secrets
+support_url: https://github.com/Willt123/bitrise-step-fetch-infisical-secrets/issues
+published_at: 2025-07-14T20:20:17.478584+09:30
+source:
+  git: https://github.com/Willt123/bitrise-step-fetch-infisical-secrets.git
+  commit: 5c6f3f41b8c5ce859a0841230de8de4e61d58368
+type_tags:
+- utility
+toolkit:
+  go:
+    package_name: github.com/Willt123/bitrise-step-fetch-infisical-secrets
+inputs:
+- infisical_client: null
+  opts:
+    description: The client ID is required to identify the machine identity from Infisical
+    is_expand: true
+    is_required: true
+    summary: The client ID from infisical.
+    title: Infisical Client ID
+- infisical_client_secret: null
+  opts:
+    description: The client secret is required to authenticate the machine identity
+      from Infisical
+    is_expand: true
+    is_required: true
+    summary: The client secret from infisical.
+    title: Infisical Client Secret
+- infisical_project_id: null
+  opts:
+    description: The project ID is required to access the correct project from Infisical
+    is_expand: true
+    is_required: true
+    summary: The project ID from infisical.
+    title: Infisical Project ID
+- infisical_env: null
+  opts:
+    description: The env is required to match the env specified in the token above
+      to provide access to the required secrets
+    is_expanded: true
+    is_required: true
+    summary: The slug for your project environment
+    title: Infisical Env
+- infisical_url: null
```

Comment:
nit: if this input has a default value, you might as well set it in this step.yml (e.g. `infisical_url: https://example.com`). This will be shown in the Workflow Editor for users.